geocity/apps/forms/models.py

```diff
@@ -724,7 +724,8 @@ class Meta:
 INPUT_TYPE_NUMBER = "number"
 INPUT_TYPE_REGEX = "regex"
 INPUT_TYPE_TEXT = "text"
-INPUT_TYPE_TITLE = "title"
+INPUT_TYPE_DISPLAY_TEXT = "text_output"
+INPUT_TYPE_DISPLAY_TITLE = "title_output"
```

Comment:
Why not use the same wording as the constants? `display_text` and `display_title`?